packages/sdk-node/src/index.ts

```diff
@@ -25,7 +25,7 @@
  */
 
 // Version 21.0.4
- export * from './nodeServices'
+export * from './nodeServices'
```

Comment:
This was reported by John from a previous PR and I said I would fix in the next PR I put up.